tests/unit/composables/Project.spec.js

```diff
@@ -396,4 +399,15 @@ describe('Test composable: Project', () => {
       expect(await rm('error').catch((e) => e)).toBeDefined();
     });
   });
+
+  describe('Test function: getStatus', () => {
+    it('should be a success', async () => {
```

Comment:
and return something, no ?

Reply:
done

Reply:
bad correction.
`return an array with one valid FileStatus`

Reply:
done